add requirements.txt, incremental updates (#6)
sbcshop/UPS-Hat-RPi: README.md

# UPS-Hat-for-Raspberry-Pi
<img src= "https://github.com/sbcshop/UPS-Hat-for-Raspberry-Pi/blob/main/Images/banner.png" />

> The UPS Hat is a 5v power supply, It fit on top of the Raspberry Pi and gives power backup if the Pi's USB power supply is cut off. It has various protections like short circuit protection, reverse battery protection and indicator, overcharge/discharge protection, and overcurrent protection. It has a 5V USB output, convenient for powering other boards.

Reviewer: ```suggestion
The UPS Hat is a 5v power supply. It fits on top of the Raspberry Pi and gives power backup if the Pi's USB power supply is cut off. It has various protections like short circuit protection, reverse battery protection and indicator, overcharge/discharge protection, and overcurrent protection. It has a 5V USB output, convenient for powering other boards.
```

<img src= "https://github.com/sbcshop/UPS-Hat-for-Raspberry-Pi/blob/main/Images/inout-voltage.png" />

## Features
  * Two on board battery reverse indicator LEDs
  * Two on board USB power port for external devices

>   * Onboard oled display, for displaying current, voltage,power and battery percentage of UPS HAT

Reviewer: ```suggestion
  * Onboard OLED display, for displaying current, voltage, power, and battery percentage of UPS HAT
```

## Code
### Setup

> 1. Enable I2C in raspberry pi. Run `sudo raspi-config` then go to "interface options->I2C->yes->press enter"

Reviewer: ```suggestion
1. Enable I2C on the Raspberry Pi. Run `sudo raspi-config` then go to "interface options->I2C->yes->press enter"
```

1. Install python libraries: `sudo -H pip3 install -r requirements.txt`

### folder contents

This folder contains two python files and Images, Fonts folders:
   * **INA219_UPS.py**  -> This file you need to run, displays the load voltage, current, power, and battery percentage on the terminal as well as in OLED
   * **oled_091.py**    -> OLED display library
   * **Images/**        -> images which we display in OLED display
   * **Fonts/**         -> various fonts, which we use in the OLED display

## Steps To Use UPS HAT (Follow These Steps) (**Important**)

> You need to follow this process when you use UPS HAT. if you use this process, your OLED problem is also resolved (static noise pattern)

Reviewer: ```suggestion
You need to follow this process when you use UPS HAT. If you use this process, your OLED problem is also resolved (the static noise pattern).
```
